telegram/error.py

```diff
@@ -139,3 +139,18 @@ class Conflict(TelegramError):
 
     def __reduce__(self) -> Tuple[type, Tuple[str]]:
         return self.__class__, (self.message,)
+
+
+class TelegramDecryptionError(TelegramError):
```

Comment:
Is moving the error related to this PR? b/c it's at least somewhat breaking (people might for some reason import it from `telegram.passport.credentials` instead of from `telegram`), so I'd prefere to do this in v14

Reply:
It wasn't related to slots, just saw that this wasn't in the right file and thought of moving it. Will revert